src/applications/gqrx/remote_control.cpp

```diff
@@ -74,35 +74,55 @@ void RemoteControl::stop_server()
 /*! \brief Read settings. */
 void RemoteControl::readSettings(QSettings *settings)
 {
-    bool conv_ok;
+    if (!settings)
+        return;
 
-    rc_freq = settings->value("input/frequency", 144500000).toLongLong(&conv_ok);
-    rc_filter_offset = settings->value("receiver/offset", 0).toInt(&conv_ok);
+    settings->beginGroup("remote_control");
```

Comment:
Although the beginGroup() / endGroup() change is useful, they don't have anything to do with the topic and should therefore be a separate commit and pull request. Including them here only makes the present pull request more difficult to understand.